pkg/cluster/install_test.go

```diff
@@ -30,6 +31,23 @@ import (
 
 func failingFunc(context.Context) error { return errors.New("oh no!") }
 
+func normalFunc(context.Context) error {
+	time.Sleep(1 * time.Second)
```

Comment:
I don't like sleeps in tests :)

Alternatively, perhaps this would be better:

runSteps takes two more arguments -- `startTime int64, currentTime func() int64` -- the former being the time the function was ran, and in the main code just `time.Now`. Then, in the test, you can have startTime be some number, and then currentTime just increment it every time it is called, and then you don't need to have a sleep :)

(alternatively, we could define some fake interface which does the same thing, that we can pass to Cluster, but I'm not sure if that's worth the effort unless we're using it elsewhere.)

Reply:
@hawkowl Hi Amber, I did it in another way, added a `metricsDryrun` param in step run funcs, if it is set to `true`, the func will set static time value for each step run